### ISSUE TITLE
Inquiry about Dataset Partitioning in Baseline Methods

### ISSUE THREAD

**Issue by Jeromeysjd** [ysmei97/DCFL#1]
Hello,I am currently working on reproducing this paper. During my replication process, I encountered some questions regarding the experimental settings of the baseline methods in your paper. Specifically, I am interested in knowing whether the dataset partitioning methods (such as the ratio of the training set to the testing set and other relevant aspects. For example, for MNIST and FashionMNIST, the training and test sets you use follow the official division, i.e. train=True for the training set and train=False for the testing set.) used for the baseline methods are consistent with those of the DCFL approach proposed in your paper.

In your paper, you have comprehensively described the dataset partitioning for DCFL in different scenarios. However, the details of dataset partitioning for baseline methods are not explicitly stated. To ensure the accuracy and comparability of my replication, I believe it is crucial to clarify this point. If the partitioning methods are the same, it would be great if you could confirm this. If there are differences, I would greatly appreciate it if you could provide some guidance on these variations.

您好。我目前正在复现您的论文，在我的复现过程中，遇到了一些关于您论文中基线方法的实验设置的问题。具体来说，我想知道数据集的划分方法（如训练集和测试集的比例以及其他相关方面）是否与您提出的DCFL方法实验中划分标准一致？例如，对于 MNIST 和 FashionMNIST，您使用的训练集和测试集是按照官方划分的，即训练集为 train=True，测试集为 train=False。

在您的论文中，您全面描述了 DCFL 在不同场景下的数据集划分。但是，基线方法的数据集划分细节没有明确说明。为了确保复现的准确性和可比性，我认为澄清这一点至关重要。如果划分方法相同，还请您确认这一点。如果存在差异，如果您能提供一些指导，我将不胜感激。
